presto-client/src/main/java/com/facebook/presto/client/QueryResults.java

```diff
@@ -332,6 +333,10 @@ private static Object fixValue(TypeSignature signature, Object value)
             case CHAR:
             case GEOMETRY:
                 return String.class.cast(value);
+            case BING_TILE:
```

Comment:
nit: 
```
// Bing tiles are serialized as strings when used as map keys/values,
// they are serialized as json otherwise (value will be a LinkedHashMap).
```